Submissions/dummygithubusername/Assignment-1/assignment-1.py

```diff
@@ -0,0 +1,55 @@
+from hashlib import sha256
```

Comment:
to use hashlib.sha256 you need to import `hashlib` completely!

import hashlib

Reply:
import hashlib
corrected.